### PR TITLE
SLING-11150: Return package id in the DistributionResponse for a dete…

### PR DESCRIPTION
**Pull request by amit-jain**
…rministic way of identifying status

- Add DistributionResponseInfo object to implementations of DistributionResponse
- Add package id in the item properties returned from the queue resource provider

### CODE REVIEW

**Comment by amit-jain** [comment]
Depends upon https://github.com/apache/sling-org-apache-sling-distribution-api/pull/10

File: pom.xml [hunk 1]
```diff
@@ -245,7 +245,7 @@
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.distribution.api</artifactId>
-            <version>0.4.0</version>
+            <version>0.5.1-SNAPSHOT</version>
```
Comment (joerghoh) [inline pom.xml]: I would prefer if this would be release version, especially when we are talking about the API.

Comment (amit-jain) [reply pom.xml]: @joerghoh Yes, that's the intention once the api PR is merged and released. Have converted to draft in the meantime.

**Review by tmaret — APPROVED** [review]

**Comment by tmaret** [comment]
SLING-11154 - Implement DistributionResponse#getDistributionInfo

**Pull request merged**